documenting/README.md

```diff
@@ -0,0 +1,10 @@
+# connector-documentation
+
+Let's automate connector documentation.
+Creates .md, and .html (optional) files automatically from info.json.
+
+Usage: 
+
+`python3 autodocs.py /path/to/info.json`
```

Comment:
how does provide md/html/both as inputs?

Reply:
Command line argument functionality is now available for file types